app/controllers/providers/merits_declarations_controller.rb

```diff
@@ -0,0 +1,35 @@
+module Providers
+  class MeritsDeclarationsController < BaseController
+    include ApplicationDependable
+    include Steppable
+
+    before_action :authorize_legal_aid_application
+
+    def show
+      @form = MeritsAssessments::MeritsDeclarationForm.new(model: merits_assessment)
+    end
+
+    def update
+      @form = MeritsAssessments::MeritsDeclarationForm.new(merits_declaration_params.merge(model: merits_assessment))
```

Comment:
I wonder if we even need a form and some params here.
Because there is no validation needed, no condition. If the user clicks on the button in the page, we always need to set `client_merits_declaration` to `true`
I reckon this would be enough:
```ruby
def update
  merits_assessment.update!(client_merits_declaration: true)
  # TO DO add correct redirect here when it is implemented
  # redirect_to next_step_url
  render plain: 'Placeholder: Merits check your answers page will go here'
end
```

And in the page you can simplify the submit button:
```ruby
<%= form.button t('generic.continue'), class: 'govuk-button' %>
```

Other devs, what do you think?

Reply:
I agree. If there is no validation, there is no need for a Form object.

In fact I think you can simplify this even further by removing the form on the page and replacing it with:
```erb
<%= button_to t('generic.continue'), path_to_action, method: :patch, class: 'govuk-button' %>
```